src/static/app/client/client-content/ex-alter-check.tag

```diff
@@ -11,16 +11,16 @@
 
     <!-- Step 1 -->
     <section if="{ !isEditing() }">
-      <h4 class="ui header">Step1. Prepare alter sql</h4>
+      <h3>Step1. Prepare alter sql</h3>
       <alter-check-checked checkedzip="{ checkedZip }" unreleaseddir="{ unreleasedDir }" />
       <alter-check-form ref="altercheckform" projectname="{ opts.projectName }" updatehandler="{ updateBegin }" />
     </section>
 
     <!-- Step 2 -->
     <section show="{ isEditing() }">
-      <h4 class="ui header">Step2. Execute AlterCheck</h4>
+      <h3>Step2. Execute AlterCheck</h3>
 
-      <h5 class="ui header">Open Editing Alter SQL Files</h5>
+      <h5>Open Editing Alter SQL Files</h5>
```

Comment:
Please use h4 after h3.

Reply:
fixed it like this:
https://github.com/dbflute/dbflute-intro/pull/231/files#diff-91510226fa32ca6037e514a4e542b333R98